### DISCUSSION TITLE
Separate Open or Closed from other types

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#993] [Ideas]
Would it be possible for a future release to make open or closed type separate from the other types with the default being open? AA meetings are either open or closed and they can’t be both (as the current plugin allows).

**Comment by tim-rohrer** [comment]
TSML hasn't allowed a meeting to be both in the past. If you're seeing that now, something else is going on. I won't be able to check it out in detail until later, but for now, please check to see if you have any type customizations that might be causing you to see both.

Please post a screen shot showing both.

**Reply by Grateful-Viv** [reply]
I see now in the edit screen that though I can check both open and closed, it won't save both after selecting update (and I assume is the case for publishing a new meeting). 

However I can save it with neither checked, and am wondering if a future release would default to open if closed wasn't checked? Recently I had to change two different meetings from closed to open, but all I did was uncheck closed (forgot to check open).

**Reply by joshreisner** [reply]
i dont think we can assume that every  administrator knows the open/closed status of every meeting.

if we start adding the open type whenever closed is not indicated we could have some unexpected results

**Reply by tim-rohrer** [reply]
Agreed. And I believe a number of areas would prefer to default to closed.
